ansible/files/admin_api_scripts/pg_upgrade_initiate.sh

```diff
@@ -0,0 +1,82 @@
+#! /usr/bin/env bash
+
+set -eEuo pipefail
+
+PGVERSION=$1
+
+MOUNT_POINT="/data_migration"
+
+run_sql() {
+    STATEMENT=$1
+    psql -h localhost -U supabase_admin -d postgres -c "$STATEMENT"
+}
+
+cleanup() {
+    UPGRADE_STATUS=${1:-"failed"}
+    EXIT_CODE=${?:-0}
+
+    systemctl start postgresql
+    run_sql "CREATE EXTENSION IF NOT EXISTS pg_graphql CASCADE;"
```

Comment:
oh that sweet special casing

Reply:
Hah, yep. It's the only extension that we use that uses regtypes, referencing system OIDs. 

Will break this out into a list of extensions and make the script iterate over that so it's easier to notice and manage. Also comments.